### ISSUE TITLE
You can accidentially comment multiple times

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#1128]
https://ldjam.com/events/ludum-dare/39/polarized

```
Tim Eriksen (@tim-eriksen)
posted 4 minutes ago
Cool game! What it lacks in graphics and a clear relation to the theme it makes up for in puzzle quality. I wish there were some better instructions to begin with, and the controls were a bit awkward as I would often push the cubes further than I was trying to. Over all: good work!
```

that got posted 3 times while the website was slow.

**Comment by Jezzamonn** [comment]
I know twitter handles this by saying you can't post the same thing twice in a row (e.g. you can't tweet the exact same tweet twice)

**Comment by povrazor** [comment]
I believe this can be reproduced by clicking publish multiple times. When the website is fast though, there usually isn't time to do this multiple times.

**Comment by djdduty** [comment]
Does spam clicking the button reproduce the same issue, then?

**Comment by local-minimum** [comment]
Maybe we should simply hash the content and refuse resubmit without change? Or perhaps better to deactivate the button right on click.

**Comment by povrazor** [comment]
Yeah this is not a complicated problem. Setting a state after you click, and unsetting it after an error

**Comment by local-minimum** [comment]
It would kind of be good if we had that for all buttons more or less though

**Comment by povrazor** [comment]
Yep. It should be a button property. There's little reason for a button to trigger twice.
